xsnap/sources/xsnap.c

```diff
@@ -2,14 +2,19 @@
 
 #define SNAPSHOT_SIGNATURE "xsnap 1"
 
+extern void fxDumpSnapshot(xsMachine* the, xsSnapshot* snapshot);
+
 static void xsBuildAgent(xsMachine* the);
-static void xsPlayTest(xsMachine* the);
 static void xsPrintUsage();
+static void xsReplay(xsMachine* machine);
 
 //static void xs_clearTimer(xsMachine* the);
+static void xs_currentMeterLimit(xsMachine* the);
 static void xs_gc(xsMachine* the);
 static void xs_issueCommand(xsMachine* the);
+static void xs_performance_now(xsMachine* the);
 static void xs_print(xsMachine* the);
+static void xs_resetMeter(xsMachine* the);
```

Comment:
Should `xs_currentMeterLimit`, `xs_performance_now`, and `xs_resetMeter` be here?

Maybe they're needed to replay stuff produced by xsnap-worker?